src/bin/vaults/CommandPermissions.ts

```diff
@@ -1,115 +0,0 @@
-// Import Logger, { LogLevel, StreamHandler } from '@matrixai/logger';
```

Comment:
Uncomment this and keep it in `CommandVaults`. Should be similar to `identities/CommandPermissions`.

Note that we can later make it work in `pk acl permissions <vaultid/identityid/nodeid>` something like that.

Reply:
Why are we doing this? all functionality for this has been stripped out or moved elsewhere.

Reply:
It's only to reimplement `pk vaults permissions` command.

Reply:
I have re implemented the vaults permissions command.